mlir/lib/Dialect/Math/Transforms/ExpandPatterns.cpp

```diff
@@ -441,6 +441,25 @@ static LogicalResult convertRoundEvenOp(math::RoundEvenOp op,
   return success();
 }
 
+// Convert `math.sqrt` into `arith.divf` + `math.sqrt`
```

Comment:
```suggestion
// Convert `math.rsqrt` into `arith.divf` + `math.sqrt`
```

Reply:
I was in for an infinite conversion loop... Thanks :)